pyproject.toml

```diff
@@ -63,6 +63,7 @@ coveralls = "^1.9"
 setuptools = ">65.5.1"
 sphinx = "6.2.1"
 sphinx_rtd_theme = "*"
+jsonschema = "<4.18.0"
```

Comment:
Probably might not need to have this limitation. Check before marking the PR ready for review.